fern/server/definition/generations.yml

```diff
@@ -16,18 +16,6 @@ service:
       path: /generations
       request: UpdateGenerationRequest
       response: commons.Observation
-    get:
```

Comment:
we could have one generic endpoint to return observations. As far as i know, this is only used in datasets. We should move it to 'observations' now as it is early.